docs/docsite/.templates/banner.html

```diff
@@ -61,14 +55,11 @@
       }
       msg += '</p>';
 
-      banner += '<div id="banner_id" class="admonition ';
-      banner += important ? 'important' : 'caution';
-      banner += '">';
+      banner += '<div id="banner_id" class="admonition '; banner += important ? 'important' : 'caution'; banner += '">';
       banner += important ? '<br>' : '';
       banner += msg;
       banner += important ? '<br>' : '';
       banner += '</div>';
       document.write(banner);
     {% endif %}
-  </script>
-{% endif %}
+</script>
```

Comment:
I'd just keep it indented to hide from the diff
```suggestion
  </script>
```